docs/02_cleos/03_command-reference/net/connect.md

```diff
@@ -1,17 +1,56 @@
+## Command
+```sh
+cleos net connect [OPTIONS] host
+```
+
+**Where:**
+* [OPTIONS] = See **Options** in the [**Command Usage**](command-usage) section below.
+* host = The hostname:port to connect to.
+
+**Note:** The arguments and options enclosed in square brackets are optional.
+
 ## Description
-Start a new connection to a peer
+Start a new connection to a specified peer. This command allows an operator's node to connect to another peer without restarting the node.
+
+## Command Usage
+The following information shows the different positionals and options you can use with the `cleos net connect` command:
+
+### Positionals
+* `host` _TEXT_ REQUIRED - The hostname:port to connect to.
```

Comment:
Full stop at end ... template indicate this list doesn't have them

Reply:
Removed.